### ISSUE TITLE
Fedora 40 after codec update black screen

### ISSUE THREAD

**Issue by YuryenUcahh** [jeffshee/gnome-ext-hanabi#132]
**Description**
I updated codecs today and clapper not working. I tried mp4, mkv and webm videos and only working mkv videos 

**OS and Shell Information**
 - OS: Fedora 40
 - GNOME Shell: 46
 - Session type: Wayland

**Additional Information**
I already installed rpm fusion codecs 
force gtkmediafile and gtk4paintablesink working fine but my cpu usage 0.8% (clapper) to 4.7% 

https://github.com/jeffshee/gnome-ext-hanabi/assets/51506997/cdded4f0-4121-473a-9a49-db10132564e5

**Comment by dominictynes** [comment]
I think I had the same issue.  I eventually figured out if I uninstall clapper it would start working.  Not sure where the root issue is.

**Comment by dominictynes** [comment]
FYI...

When I try Hanabi with clapper from fedora repo (clapper-0.5.2-6.fc40.src.rpm) I see the same issue you described.  
Even running that clapper version as a standalone video player (nothing to do with Hanabi) gives me an error:
(com.github.rafostar.Clapper:89493): Gjs-CRITICAL **: 23:00:51.686: JS ERROR: Gio.IOErrorEnum: Error sending credentials: Error sending message: Broken pipe

But if I install clapper from the rawhide repo (clapper-0.6.0-1.fc41.src.rpm) Hanabi runs fine again.

**Comment by YuryenUcahh** [comment]
> FYI...
> 
> When I try Hanabi with clapper from fedora repo (clapper-0.5.2-6.fc40.src.rpm) I see the same issue you described. Even running that clapper version as a standalone video player (nothing to do with Hanabi) gives me an error: (com.github.rafostar.Clapper:89493): Gjs-CRITICAL **: 23:00:51.686: JS ERROR: Gio.IOErrorEnum: Error sending credentials: Error sending message: Broken pipe
> 
> But if I install clapper from the rawhide repo (clapper-0.6.0-1.fc41.src.rpm) Hanabi runs fine again.

yes clapper-0.6.0-1.fc41.src.rpm works but at the same time cpu usage has increased

Experimental VA plugin not working now 

![Ekran Görüntüsü - 2024-06-15 10-22-38](https://github.com/jeffshee/gnome-ext-hanabi/assets/51506997/53c46516-47ab-49bf-af46-d8eb0e0001c0)
![Ekran Görüntüsü - 2024-06-15 10-33-40](https://github.com/jeffshee/gnome-ext-hanabi/assets/51506997/7b291c38-dc05-419c-894d-c06152aa861a)

**Comment by jeffshee** [comment]
Upstream Clapper packaging issue — it fails as a standalone player too, per @dominictynes. Clapper is optional now anyway; Hanabi defaults to `gtk4paintablesink`.
